.gitlab-ci.yml

```
@@ -233,6 +233,14 @@ cargo-deny:
   # FIXME: Temorarily allow to fail.
   allow_failure:                   true
 
+cargo-fmt:
+  stage:                           test
```

as a debug step, just to avoid running everything else, move it to the previous stage
```suggestion
  stage:                           .pre
```
just do not forget to revert this at the end :)

---

Ty, but that works ;)